meetings/2020-07-16.md

```diff
@@ -0,0 +1,93 @@
+﻿# Node.js Technical Steering Committee (TSC) Meeting 2020-07-16
+
+## Links
+
+
+* **Recording**:  https://youtu.be/yVMhtWJDl-Q
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/893
+
+## Present
+
+
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Tobias Nießen @tniessen (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+
+* We have three new collaborators:
+  - https://github.com/nodejs/node/issues/34100
+  - https://github.com/nodejs/node/issues/34099
+  - https://github.com/nodejs/node/issues/34098
+* and ~~four~~ three further nominations that have been accepted:
+  - https://github.com/nodejs/node/issues/34121
+  - https://github.com/nodejs/node/issues/34168
+  - https://github.com/nodejs/node/issues/34242
+  - ~~https://github.com/nodejs/node/issues/34241~~
+* plus one pending:
+  - https://github.com/nodejs/node/issues/34369
+
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* FYI - changes to Foundation bylaws approved recently and published. See https://github.com/openjs-foundation/cross-project-council/issues/590 for some info about changes.
+
+### nodejs/node
+
+* Next 10 years of Node.js  [#34260](https://github.com/nodejs/node/issues/34260)
+  * FYI - on agenda as FYI, first session today.
+
+* Proposal: create a triage team to manage the backlog [#34054](https://github.com/nodejs/node/issues/34054)
+  * Gireesh has action to submit PRs to add to governance to further discussion. We’ll remove
+    the agenda tag for now

```

Comment:
btw, I have the PR in place, ready to review: https://github.com/nodejs/node/pull/34295